README.md

```diff
@@ -82,12 +82,12 @@ const writer = new CssWriter([
 
 ### Consuming content from the writer
 
-The writer is a readable stream in object mode so in order to access the data
-you may register a data handler and listen for objects to be passed to the
-handler:
+The writer is an event emitter, which has a method called `bundle`, which
+returns a readable stream in object mode so in order to access the data you may
+register a data handler and listen for objects to be passed to the handler:
```

Comment:
Is this right? Shouldn't bundle return a stream that outputs string chunks of bundled css?
I thought that the js writer was a feed stream but if you call .bundle() then you get a stream that streams the bundled js text?

Reply:
We discussed this IRL